vaadin-dev-server/src/main/resources/com/vaadin/base/devserver/dev-mode-not-ready.html

```diff
@@ -54,7 +54,7 @@
 const poll = () => {
   try {
     fetch(window.location.href, { headers: { 'X-DevModePoll': 'true' } }).then(response => {
```

Comment:
Needs an `await` to catch async errors in the outer `try ... catch`:
```suggestion
    await fetch(window.location.href, { headers: { 'X-DevModePoll': 'true' } }).then(response => {
```

Reply:
There is not top level await support in all Vaadin 23 browsers, so maybe `.catch()` is better then

Reply:
ah but this isn't even top level...

Reply:
Maybe the new version makes more senes